### DISCUSSION TITLE
TSC Meeting October 2020

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7386] [Team Posts]
**General**
Date (end of October) TBD.

**Membership**
Formalise the membership structure, this has been on the back burner for a while so lets have something simple, balanced and formal that should work for everyone. I suggest something borrowed from other similar open source projects. More details to follow (open to suggestions of good governance models from other projects).

**Technical Agenda**
1. v1.6 release.
2. Zephyr Support and plans.
3. v1.7 release features and schedule.
4. Intel plans for improving developer tooling (FDK)
5. SOF driver updates and forthcoming plans.
6. Improvements to topology ALSA conf.
7. Fuzzing GSoC results and future plans
8. Code-ql integration, static analysis
9. GSoC 2021: Replacement for M4?
10. GSoC 2021 IMX: switch to zephyr, enable i.MX RT
11. More ?? (please edit and add).

**Any other business**
1) Please edit and add.

**Comment by lgirdwood** [comment]
@thesofproject/steering-committee fyi - please edit to add topics and detail.

**Comment by lgirdwood** [comment]
@ryans-lee fyi - for Maxim

**Comment by cujomalainey** [comment]
I would double check Ryan got that ping as he is not a member of this group.

**Comment by cujomalainey** [comment]
October is almost half over, we should probably pick a date

**Comment by lgirdwood** [comment]
@cujomalainey just back from vacation - let  me sort this out today. I will send out a calendar so that folks can choose. Will include first week in November.

**Comment by lgirdwood** [comment]
@dbaluta would you be able to setup an doodle calendar for the TSC meeting for Europe afternoons and US mornings from 26th October to Nov 6th. I didn't realise doodle was a paid service and don't have an account. Thanks

**Comment by lgirdwood** [comment]
@dbaluta I can send the email if you can paste the link here.

**Comment by dbaluta** [comment]
@lgirdwood Please use this link: https://doodle.com/poll/qgdwsx83vu5umhsu
